CONTRIBUTING.md

```diff
@@ -101,6 +101,10 @@ if any existing recipes cover their creation.
    ```
    git checkout -b <new-branch-name> develop
    ```
+   or, if that doesn't work:
+   ```
+   git checkout -b <new-branch-name> origin/develop
+   ```
 2. Fetch the latest changes from the remote `develop` branch:
```

Comment:
```suggestion

This alternative can be useful in cases where your local develop branch is out of sync with the remote repository, or if you want to ensure that your new branch is based on the most recent changes from the remote repository.

2. Fetch the latest changes from the remote `develop` branch:
```

I'd add just a little bit more context. The differences between the commands are subtle, but worth spelling out.